src/app_service/service/community/dto/community.nim

```diff
@@ -283,6 +290,11 @@ proc toCommunityTokenPermissionDto*(jsonObj: JsonNode): CommunityTokenPermission
   if (tokenPermissionTypeInt >= ord(low(TokenPermissionType)) or tokenPermissionTypeInt <= ord(high(TokenPermissionType))):
       result.`type` = TokenPermissionType(tokenPermissionTypeInt)
 
+  var tokenPermissionStateInt: int
+  discard jsonObj.getProp("state", tokenPermissionStateInt)
+  if (tokenPermissionStateInt >= ord(low(TokenPermissionState)) or tokenPermissionStateInt <= ord(high(TokenPermissionState))):
```

Comment:
```suggestion
  if (tokenPermissionStateInt >= ord(low(TokenPermissionState)) and tokenPermissionStateInt <= ord(high(TokenPermissionState))):
```

Reply:
Nice catch, I blindly copied it from the line above :sweat_smile:

Reply:
Fixed